easybuild/easyconfigs/q/Qt/Qt-5.2.1-intel-2015a.eb

```diff
@@ -0,0 +1,21 @@
+easyblock = 'ConfigureMake'
+
+name = 'Qt'
+version = '5.2.1'
+
+homepage = 'http://qt-project.org/'
+description = "Qt is a comprehensive cross-platform C++ application framework."
+
+toolchain = {'name': 'intel', 'version': '2015a'}
+
+source_urls = [
+    'http://download.qt.io/archive/qt/%(version_major_minor)s/%(version)s/single/'
+]
+sources = ['%(namelower)s-everywhere-opensource-src-%(version)s.tar.gz']
+
+configopts = "-static -qt-xcb -release -opensource -confirm-license -c++11 -no-qml-debug -no-gif -no-libpng -no-libjpeg"
+configopts += " -no-freetype -no-harfbuzz -no-openssl -no-xinput2 -no-xcb-xlib -no-widgets -no-nis -no-cups -no-dbus"
+configopts += " -no-eglfs -no-directfb -no-linuxfb -no-kms -no-opengl -nomake tools -nomake examples"
+configopts += " -platform linux-icc-64"
```

Comment:
we have a Qt easyblock already, so you should update it to use these configure options in a block specific to Qt v5.x

for several of them, you probably need alternatives in case the required dependencies *are* provided (cfr. the Qt easyconfig in #1375)